crates/bgp-pkt/src/path_attribute/bgp_sid.rs

```diff
@@ -0,0 +1,131 @@
+use crate::{
+    iana::BgpSidAttributeType, nlri::MplsLabel, path_attribute::PathAttributeValueProperties,
+};
+use serde::{Deserialize, Serialize};
+
+/// 3 bytes MPLS Label
+/// followed by
+/// 3 bytes for range size
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[cfg_attr(feature = "fuzz", derive(arbitrary::Arbitrary))]
+pub struct SRGB {
```

Comment:
I would rename to `SegmentRoutingGlobalBlock` or `SrGlobalBLock` and maybe take the definition from RFC8402 for in addition to the just description bytes layout doc